Documentation/ceph-osd-mgmt.md

```diff
@@ -63,23 +60,51 @@ through the removal process:
+   - For example if the device set is named `set1` and the `count` was reduced from `3` to `2`, the orphaned PVC would have the index `2` and might be named `set1-2-data-vbwcf`
+3. Identify the orphaned OSD.
+   - The OSD assigned to the PVC can be found in the labels on the PVC
+   - `kubectl -n rook-ceph get pod -l ceph.rook.io/pvc=<orphaned-pvc> -o yaml | grep ceph-osd-id`
+   - For example, this might return: `ceph-osd-id: "0"`
+
+### Confirm the OSD is down
+
+If you want to remove a unhealthy OSD, the osd pod may be in an error state such as `CrashLoopBackoff` or the `ceph` commands
+in the toolbox may show which OSD is `down`. If you want to remove a healthy OSD, you should run
+`kubectl -n rook-ceph scale deployment rook-ceph-osd-<ID> --replicas=0` and `ceph osd down osd.<ID>` from the toolbox.
 
-1. Determine the OSD ID for the OSD to be removed. The osd pod may be in an error state such as `CrashLoopBackoff` or the `ceph` commands
-in the toolbox may show which OSD is `down`. If you want to remove a healthy OSD, you should run `kubectl -n rook-ceph scale deployment rook-ceph-osd-<ID> --replicas=0` beforehand.
-2. Mark the OSD as `out` if not already marked as such by Ceph. This signals Ceph to start moving (backfilling) the data that was on that OSD to another OSD.
+### Purge the OSD from Ceph cluster
+
+OSD removal can be automated with the example found in the [rook-ceph-purge-osd job](/cluster/examples/kubernetes/ceph/osd-purge.yaml).
+You can go to [Delete the underlying data][#delete-the-upderlying-data] section after running this job. If you want to remove OSD by hand,
+please read the following sections. However, we recommend you to use the above-mentioned job to avoid operation errors.
+
+Run the following commands from the toolbox pod.
+
+1. Mark the OSD as `out` if not already marked as such by Ceph. This signals Ceph to start moving (backfilling) the data that was on that OSD to another OSD.
    - `ceph osd out osd.<ID>` (for example if the OSD ID is 23 this would be `ceph osd out osd.23`)
-3. Wait for the data to finish backfilling to other OSDs.
+2. Wait for the data to finish backfilling to other OSDs.
    - `ceph status` will indicate the backfilling is done when all of the PGs are `active+clean`. If desired, it's safe to remove the disk after that.
-4. Update your CephCluster CR such that the operator won't create an OSD on the device anymore.
-Depending on your CR settings, you may need to remove the device from the list or update the device filter.
-If you are using `useAllDevices: true`, no change to the CR is necessary.
-5. Remove the OSD from the Ceph cluster
+3. Remove the OSD from the Ceph cluster
    - `ceph osd purge <ID> --yes-i-really-mean-it`
-6. Verify the OSD is removed from the node in the CRUSH map
+4. Verify the OSD is removed from the node in the CRUSH map
    - `ceph osd tree`
 
-### Remove the OSD Deployment
+### Remove the Kubernetes resources
```

Comment:
This is not necessary if they used the osd purge job, so let's keep this in the manual section, no need for the separate title.
```suggestion

```

Reply:
OK, removed.